packages/sync-service/dev/docker-compose.yml

```diff
@@ -31,3 +31,21 @@ services:
       - "host.docker.internal:host-gateway"
     depends_on:
       - postgres
+
+  # We can make these services optional, but don't care for dev
```

Comment:
Strikes me that they should still be optional, easier through a separate docker compose (that can ahem, compose with this one) or enables by a flag / env var.

Just because why pull and why have the noise if you just want to run the thing and aren’t interested in telemetry for what you’re doing.

Reply:
yeah, there are a few options. 1. Can also use profiles so that we don't have multiple compose files; 2. another compose for services+telemetry.